From 3cea511bbf4262c096eb625e5f5040a5364d21c2 Mon Sep 17 00:00:00 2001
From: Igal Liberman <igall@marvell.com>
Date: Wed, 1 Feb 2017 11:36:02 +0200
Subject: [PATCH 1353/2241] [2/2]: dts: apn806: remove revision-info node

The SoC information driver was used to distinguish between
A8040/7040 A0 and A1. A0 is no longer supported, the driver
was disabled in a separate patch so the node can be removed
from the device-tree.

Change-Id: Ia79b52d35f45be0d9fd45de3d2986e93c1c08ca3
Signed-off-by: Igal Liberman <igall@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/36315
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-by: Hanna Hawa <hannah@marvell.com>
---
 arch/arm64/boot/dts/marvell/armada-ap806.dtsi | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/arch/arm64/boot/dts/marvell/armada-ap806.dtsi b/arch/arm64/boot/dts/marvell/armada-ap806.dtsi
index 1d4c68e..1f63ff2 100644
--- a/arch/arm64/boot/dts/marvell/armada-ap806.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-ap806.dtsi
@@ -435,13 +435,7 @@
 				interrupts = <GIC_SPI 2 IRQ_TYPE_LEVEL_HIGH>;
 				timeout-sec = <30>;
 			};
-
-			revision-info@610fcc {
-				compatible = "marvell,ap806-rev-info";
-				reg = <0x610FCC 0x4>;
-			};
 		};
 	};
-
 };
 
-- 
2.7.4

